lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
```

Comment:
I updated it to use that command in other locations as well and provide more comment on the single occurence where that helper is defined. It might even be that in the future we need to extend the function - at the single location - to do more than just a single 'up'. As @dzedro said he does not mind if @asmorodskyi still disagrees I will pull out that change and keep it for later but that will cost some minutes :-)